lib/render_window.js

```diff
@@ -54,10 +72,37 @@ class RenderWindow {
     this[RAW_RENDER_WINDOW].drawDrawable(content._raw());
   }
 
+  drawAsync(content) {
+    if (content instanceof Drawable) {
+      return new Promise((resolve, reject) => {
```

Comment:
I caught this error when test `render_window_display_drawable_async.js`:

```sh
/Users/.../Workspace/github/node-sfml/lib/render_window.js:79
          err ? reject(new Error(err)) : resolve(res);
                       ^

Error: Closed window.
    at /Users/bytedance/Workspace/github/node-sfml/lib/render_window.js:79:24
```

Reply:
This is normal because the event look is not exiting when closing the window so the next draw will throw that the window is closed ^^

Reply:
Do we have any elegant way to deal with this situation?

Reply:
Maybe we can add an `code` to the `Error` thrown to tell people whether it's an error that occurred by window closed.